### PR TITLE
Revert NWjs for Linux only

### PR DESCRIPTION
**Pull request by haslinghuis**
Mike on Slack did report an issue:

After updating NWjs `0.54.1` to `0.60.0` it seems OSD elements can't be moved on Linux platform.
We need NWjs >= `0.59.x` for Windows users so we can't simply revert to `0.54.1`

So this is a temporary fix until we have a better solution.

See https://github.com/betaflight/betaflight-configurator/pull/2797 for more details

Note: With Node 16 it's working with NWjs 0.62 ( not with 0.60 ), but there is still a big area appearing while selecting and moving an element.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by mituritsyn** [comment]
> Do you want to test this code? Here you have an automated build: [Betaflight-Configurator-Debug-Android](https://github.com/betaflight/betaflight-configurator/suites/5654975327/artifacts/185124102) [Betaflight-Configurator-Debug-Linux](https://github.com/betaflight/betaflight-configurator/suites/5654975327/artifacts/185124103) [Betaflight-Configurator-Debug-macOS](https://github.com/betaflight/betaflight-configurator/suites/5654975327/artifacts/185124104) [Betaflight-Configurator-Debug-Windows](https://github.com/betaflight/betaflight-configurator/suites/5654975327/artifacts/185124105) WARNING: It may be unstable and result in corrupted configurations or data loss. Use only for testing!

osd elements relocation works fine in portable version under Fedora OS

**Review by limonspb — APPROVED** [review]

**Comment by schugabe** [comment]
I can also confirm that that PR fixes the issue on ubuntu

**Comment by limonspb** [comment]
@haslinghuis could you please point me what is going to happen if we rever it for Windows too?

**Comment by haslinghuis** [comment]
@limonspb please read the link in the opening of this PR.

**Review by ctzsnooze — APPROVED** [review]

**Review by Asizon — APPROVED** [review]

**Pull request merged**
